app/services/course/assessment/question/programming/cpp/cpp_package_service.rb

```diff
@@ -0,0 +1,219 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::Programming::Cpp::CppPackageService < \
+  Course::Assessment::Question::Programming::LanguagePackageService
+  AUTOGRADE_INCLUDE_PATH = File.join(File.expand_path(File.dirname(__FILE__)),
```

Comment:
These specify the path to the files used in each language package service, which also depend on the package structure and filenames, which are different for each service. Will be better to keep it within the class itself.